test/Microsoft.AspNet.Mvc.ModelBinding.Test/project.json

```diff
@@ -5,6 +5,8 @@
     "dependencies": {
         "Microsoft.AspNet.FileProviders": "1.0.0-*",
         "Microsoft.AspNet.Http.Core": "1.0.0-*",
+        "Microsoft.AspNet.Mvc":"6.0.0-*",
+        "Microsoft.AspNet.Mvc.Core":"6.0.0-*",
```

Comment:
avoid these dependencies.  tests of the lower-level ModelBinding project should not bring in the dependent Mvc or Mvc.Core projects.  if there's something to test here, do it in Mvc.Core.Test or Mvc.Test

side note: Mvc will automatically bring in Mvc.Core